src/main/resources/templates/cypher/form.html

```diff
@@ -1,118 +1,139 @@
+            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+            crossorigin="anonymous"></script>
     <title>CypherDetail</title>
 </head>
 <body>
-    <div th:insert="fragments/header :: header(cypher)"></div>
-    <div class="container">
-        <div class="row">
-            <div class="col-sm-12">
-                <h2><span th:text="${heading}">Nadpis</span></h2>
-            </div>
+<div th:insert="fragments/header :: header(cypher)"></div>
+<div class="container">
+    <div class="row">
+        <div class="col-sm-12">
+            <h2><span th:text="${heading}">Nadpis</span></h2>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-sm-6">
+            <form action="#" th:action="@{/admin/cypher/add}" th:object="${cypher}" method="post">
+                <div class="form-group">
+                    <label for="cypherId">Id</label>
+                    <input id="cypherId" class="form-control" th:field="*{cypherId}" readonly/>
+                </div>
+                <div class="form-group">
+                    <label for="name">Jméno</label>
+                    <input id="name" class="form-control" th:field="*{name}"/>
+                </div>
+                <div class="form-group">
+                    <label for="stage">Pořadí</label>
+                    <input id="stage" class="form-control" type="number" th:field="*{stage}"/>
+                    <p th:if="${#fields.hasErrors('stage')}" th:errors="*{stage}"
+                       style="color: orangered; font-size: 14px">Stage error.
+                    </p>
+                </div>
+                <div class="form-group">
+                    <label for="location">Souřadnice</label>
+                    <input id="location" class="form-control" type="text" th:field="*{location}" readonly/>
+                    <small id="locationHelp" class="form-text text-muted">Kliknutím do mapy nastavíte souřadnice.
+                    </small>
+                    <p th:if="${#fields.hasErrors('location')}" th:errors="*{location}"
+                       style="color: orangered; font-size: 14px">Location error.
+                    </p>
+                </div>
+                <div class="form-group">
+                    <label for="codeword">Odpověď</label>
+                    <input id="codeword" class="form-control" type="text" th:field="*{codeword}"/>
+                </div>
+                <div class="form-group">
+                    <label for="trapCodeword">Falešná odpověď</label>
+                    <input id="trapCodeword" class="form-control" type="text" th:field="*{trapCodeword}"/>
+                </div>
+                <div class="form-group">
+                    <label for="bonusContent">Bonusové informance</label>
+                    <input id="bonusContent" class="form-control" type="text" th:field="*{bonusContent}"/>
```

Comment:
Tady radsi pouzij misto `<input>` element `<textarea>`. To same plati pro `placeDescription`. Misto radku to zobrazi takovy box, kde se lepe pise vetsi mnozstvi textu ;)

Reply:
hotovo